packages/react-components/src/Modal.tsx

```diff
@@ -19,7 +19,9 @@ interface ModalProps extends BareProps {
 }
 
 interface ActionsProps extends BareProps {
+  cancelLabel?: string;
```

Comment:
`cancelLabel?: React.ReactNode`

Having said that, I am not sure I would like this override, prefer the simplicity of consistency.